src/Player.js

```diff
@@ -0,0 +1,22 @@
+function Player() {
```

Comment:
This should not be in here :). IT's especially confusing since it's a bowling app that has players.